### PR TITLE
Batch SuperRes Example Processor

### PR DESCRIPTION
**Pull request by ProtoByter**
This contains a updated README.md file and a python file that repeatedly calls `python super_resolve.py` and all the required parameters and an extra requirements file for pip. It 1) Takes in a parameter from `--image` that tells it where the image you want to test the Super Resolution AI with, a parameter from the `--output_filename` that tells it what text to put in front of _i_.png,  `--StartAt`  allows you to start at any location you want this can be good for eg. if you turn of your computer for the night then carry on the next morning. Finally, `--Step` allows you to change the step amount on the for loop 
2) Scan the folder it is in to find any model_epochx.pth files and appends the to a list
3) Using Natsort(in the extrarequirements.txt file), it then sorts the list so that it goes from model_epoch_1.pth to the file with the biggest number
4) Calls `python super_resolve.py --input_image `(--image parameter)` --model `(using the default naming scheme for the model *.pth files)` --output_filename `(the --output_filename parameter and then the cycle the for loop is in + 1)`.png`
5) Prints how long it took, the output filename, the average time and the estimated time of finishing
6) Repeats 4 and 5 until all the model *.pth files are used

### CODE REVIEW

File: super_resolution/README.md [hunk 1]
```diff
@@ -30,3 +30,6 @@ This example trains a super-resolution network on the [BSD300 dataset](https://w
 
 ### Super Resolve
 `python super_resolve.py --input_image dataset/BSDS300/images/test/16077.jpg --model model_epoch_500.pth --output_filename out.png`
+
+### Batch Super Resolve
+`python batchsuperres.py --image dataset/BSDS300/images/test/16077.jpg`
```
Comment (msaroufim) [inline super_resolution/README.md:35]: Where should people get this image?

File: super_resolution/extrarequirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+natsort
```
Comment (msaroufim) [inline super_resolution/extrarequirements.txt:1]: need a readme to tell people to install this

File: super_resolution/batchsuperres.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import subprocess, os, fnmatch, time, datetime, argparse
```
Comment (msaroufim) [inline super_resolution/batchsuperres.py:1]: the format you're using is very different from any of our other examples, I'd suggest you take a look at them and see how your code can mimic it structurally

**Review by msaroufim — CHANGES_REQUESTED** [review]
I'd suggest you take a look at our existing examples to make your contribution look similar

**Comment by ProtoByter** [comment]
I forgot about this PR lmfao, I just saw a bunch of emails from GitHub and I was like wait wtfI honestly can't remember why I did this :P

**Pull request closed without merging**